<robot name="15_SLBNR6">
  <link name="baseLink">
    <contact>
      <lateral_friction value="1"/>
    </contact>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="1.0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
    <visual>
      <origin rpy="1.5707 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/root/catkin_ws/src/o2as_parts_description/meshes/15_SLBNR6.dae" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="1.5707 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/root/catkin_ws/src/o2as_parts_description/meshes/15_SLBNR6.vhacd.dae" scale="0.001 0.001 0.001"/> -->
      </geometry>
    </collision>
  </link>
</robot>
